fe1-web/src/features/evoting/components/EventElection.tsx

```diff
@@ -37,55 +32,21 @@ const styles = StyleSheet.create({
   } as TextStyle,
 });
 
-const EventElection = (props: IPropTypes) => {
+const EventElection: FunctionComponent<IPropTypes> = (props) => {
```

Comment:
This was recently brought to my attention
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/
https://github.com/facebook/create-react-app/pull/8177

Also, in general, the codebase is not relying on the explicit `FunctionComponent` typing.

Reply:
Interesting link! :) The only time I encountered issues when not using it is when you pass react components to other components but since this is not something we need to do here I am happy to remove it.